rector.php

```diff
@@ -15,7 +20,18 @@
 
     $parameters->set(Option::SKIP, [
         CountOnNullRector::class,
+
+        // for PHP 8
+        RemoveUnusedPromotedPropertyRector::class,
+
+        // buggy when remove private property, but filled by construct in the middle of parameter
+        RemoveUnusedPrivatePropertyRector::class,
+
+        RemoveUnusedPrivateMethodRector::class => [
+            __DIR__ . '/src/Boot/src/Bootloader/ConfigurationBootloader.php',
```

Comment:
keep private method `ConfigurationBootloader::configManager()` on purpose as it used in singleton:

https://github.com/spiral/framework/blob/cc6cb37254e8185561f12c9e6d1c0ade824ff9a3/src/Boot/src/Bootloader/ConfigurationBootloader.php#L33